packages/neo-one-client/src/__tests__/provider/neoone/JSONRPCClient.test.js

```diff
@@ -0,0 +1,161 @@
+/* @flow */
+import BigNumber from 'bignumber.js';
+
+import { keys, transactions } from '@neo-one/client-core/src/__data__';
```

Comment:
Yea, typically not good to assume the structure of an external module. Instead, let's copy the `keys` file to the `__data__` directory inside `neo-one-client`. It's definitely duplication, but in this case it's a bit cleaner than trying to import like this.